security/s2a/internal/crypter/expander.go

```diff
@@ -12,3 +14,24 @@ type hkdfExpander interface {
 	// in number of bytes, and returns the resulting expanded key.
 	expand(hash func() hash.Hash, secret, label []byte, length int) ([]byte, error)
 }
+
+// defaultHKDFExpander is the default HKDF expander which uses Go's crypto/hkdf
```

Comment:
The use of the term "default" implies that there might be a non-default HKDF expander, but I don't think that's the case. Maybe call it expander?

Reply:
If we ever use a separate crypto library for HKDF expansion like @matthewstevenson88 mentioned, I think it'll probably be called `<library_name>HKDFExpander`. Keeping this name as `expander` might be confusing if we add these separate crypto libraries later. I called it default because it uses Go's own crypto library for HKDF expansion.  WDYT?

Reply:
SG, thanks for the explanation.